cqf-fhir-cr/src/test/resources/org/opencds/cqf/fhir/cr/measure/r4/bundle/test-bundle.json

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
Issue: This content is much bigger than it needs to be. We only need the following resource:

```
{
   "resourceType": "Library",
   "id": "LibraryEvaluationTest"
} 
```

In fact, there's no reason to create a JSON resource at all. It's so simple it could just be part of the Java code.